templates/scripts/setup-user-access.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+#
+# This script is to help user to create local AWS and Kubernetes configurations
+# 
+# You should have the following information ready before running:
+# 1. your AWS user name
+# 2. your AWS access key
+# 3. your user role
+
+IAM_USERNAME=$1
+ROLE=$2
+ENVIRONMENT=$3
+
+# templated variables
+AWS_ACCOUNT_ID=<% index .Params `accountId` %>
+PROJECT=<% .Name %>
+REGION=<% index .Params `region` %>
+
+# common functions
+function usage() {
+  echo "  Usage:"
+  echo "    $0 <iam_username> <role> <environment>"
+  exit 1
+}
+
+function command_exist() {
+  command -v ${1} >& /dev/null
+}
+
+function confirm_aws_profile() {
```

Comment:
If they are only called once and it's just a single line it doesn't make much sense to make them functions.. Just for cleanliness?

Reply:
did use it more than one time...

Reply:
let's keep it